toolkit/tools/scheduler/scheduler.go

```diff
@@ -32,9 +33,11 @@ const (
 // Unlike BuildChannels, schedulerChannels holds bidirectional channels that
 // only the top-level scheduler should have. BuildChannels contains directional channels.
 type schedulerChannels struct {
-	Requests chan *schedulerutils.BuildRequest
```

Comment:
I think you've got a "leak" from other changes.;)

Reply:
Thanks, fixed it.